cspell.yaml

```diff
@@ -43,6 +43,7 @@ words:
   - localizable
   - mbps
   - mgmt
+  - mqtt
```

Comment:
Edit this cspell.yaml file instead: https://github.com/Azure/azure-rest-api-specs/blob/ba38e0686d459011f11e3bda35664c448d97fdf0/specification/deviceregistry/cspell.yaml

See updated instructions here: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#spell-check